app/models/user.rb

```diff
@@ -1,11 +1,13 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
- [ ] As we want (new) users to able to confirm their email kindly ensure that `confirmable` is enabled on this app.